devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy

```diff
@@ -0,0 +1,187 @@
+/**
+Copyright Sinopé Technologies
+1.1.1
+SVN-571
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+**/
+
+metadata {
+    preferences {
+        input("trace", "bool", title: "Trace", description: "Set it to true to enable tracing")
+    }
+
+    definition (name: "SP2600ZB-SP2610ZB Sinope Outlet", namespace: "Sinope Technologies", author: "Sinope Technologies",  ocfDeviceType: "oic.d.smartplug") 
+    {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Switch"
+		capability "Health Check"
+        
+
+        fingerprint manufacturer: "Sinope Technologies", model: "SP2600ZB", deviceJoinName: "Sinope SP2600ZB Smart Plug"

```

Comment:
Please change definition of name to `deviceJoinName: "Sinope Outlet"` in both fingerprints